README.md

```diff
@@ -1,46 +1,50 @@
-# Home assistant Sun card
-Home assistant Sun card based on Google weather design
+
+# Lovelace Sun Card: Google Weather-Inspired Design
+Elevate your Home Assistant dashboard with the beautifully designed Sun Card, inspired by Google Weather.
+This custom card displays sun-related information in a visually appealing and easy-to-read format.
```

Comment:
I suggest to add a section and stress the fact that this is an actively developed fork of the original archived project.